Marlin/src/gcode/calibrate/G34_M422.cpp

```diff
@@ -241,7 +251,7 @@ void GcodeSuite::G34() {
         linear_fit_data lfd;
         incremental_LSF_reset(&lfd);
         LOOP_L_N(i, NUM_Z_STEPPER_DRIVERS) {
-          SERIAL_ECHOLNPAIR("PROBEPT_", ('0' + i), ": ", z_measured[i]);
+          SERIAL_ECHOLNPAIR("PROBEPT_", int(i), ": ", z_measured[i]);
```

Comment:
Other recent changes suggest that a zero-based index is preferred with the `PROBEPT_` prefix.

Reply:
M422 needs the same treatment then iirc. On mobile so can't look atm

Reply:
n/p This is `M422`.

Reply:
With M422 you are specifying numbers corresponding to each Z stepper... Since the steppers are a 1-based index it makes sense to me for the M422 arguments to be 1-based.